doc/releasenotes/16_0_0_summary.md

```diff
@@ -53,3 +53,37 @@ is now fixed. The full issue can be found [here](https://github.com/vitessio/vit
 ### Deprecations
 
 The V3 planner is deprecated as of the V16 release, and will be removed in the V17 release of Vitess.
+
+### MySQL Compatibility
+
+#### Transaction Isolation Level
+Support added for `set [session] transaction isolation level <transaction_characteristic>`
+```json
```

Comment:
You should use \```sql instead of \```json

Reply:
I have not pushed the changes, this is for my tracking I have resolved that I have made the changes locally.